addons/google_calendar/views/res_users_views.xml

```diff
@@ -8,7 +8,7 @@
             <field name="arch" type="xml">
                 <notebook colspan="4" position="inside">
                     <page string="Calendar" name="calendar" groups="base.group_system">
-                        <group>
+                        <group class="word_break">
```

Comment:
You can use the bootstrap class `.text-break`[0], this way we don't have to introduce custom CSS :-)

[0]: https://getbootstrap.com/docs/4.3/utilities/text/#word-break

Reply:
Thanks for the advice :)